src/subject_reduction.ml

```diff
@@ -0,0 +1,43 @@
+open Terms
+open Print
+open Basics
+
+module Eset = Set.Make(
+  struct
+    type t = (term * term) 
+    let compare = Pervasives.compare
```

Comment:
Note that `Pervasives.compare` will not work to compare terms, since their representation contains closures (in Bindlib Binders).